test/TestGrains/StatelessWorkerGrain.cs

```diff
@@ -13,9 +10,11 @@
 
 namespace UnitTests.Grains
 {
-    [StatelessWorker(1)]
+    [StatelessWorker(ExpectedMaxLocalWorkers)]
     public class StatelessWorkerGrain : Grain, IStatelessWorkerGrain
     {
+        public const int ExpectedMaxLocalWorkers = 1;
```

Comment:
nit: MaxLocalWorkers?  'Expected' makes it sound questionable, which is valid in the testing but could be misleading in the grain implementation.